src/components/cards/CommentCardB.tsx

```diff
@@ -240,14 +240,19 @@ export const ContributorImageWrapper: React.FC<{
     salt: string;
 }> = ({ content, salt }) => {
     const contributor = getContributor(content);
-    const profilePic = contributor.properties.contributorLargeImagePath || null;
+    if (contributor) {
```

Comment:
What about an early return instead? I.e. do the check the other way round:

    if (!contributor) {
      return null;
    }

    ...

That way, the main body of the function stays simpler. I *think* TS is clever enough to support this but not sure.

Reply:
Cool - TS is fine with early returns, so I've made the change. I considered both options initially and went the other way but agree the early returns keeps the function body neater.

Reply:
Great, sounds good!